pom.xml

```diff
@@ -34,31 +34,31 @@
   </licenses>
 
   <properties>
-    <rmb.version>31.1.5</rmb.version>
-    <folio-di-support.version>1.1.0</folio-di-support.version>
-    <folio-service-tools.version>1.6.0</folio-service-tools.version>
-    <folio-holdingsiq-client.version>1.10.3</folio-holdingsiq-client.version>
-    <folio-liquibase-util.version>1.0.0</folio-liquibase-util.version>
-    <mod-configuration-client.version>4.0.3</mod-configuration-client.version>
+    <rmb.version>32.1.0</rmb.version>
+    <folio-di-support.version>1.3.0</folio-di-support.version>
+    <folio-service-tools.version>1.6.1</folio-service-tools.version>
+    <folio-holdingsiq-client.version>1.10.4-SNAPSHOT</folio-holdingsiq-client.version>
```

Comment:
Is my understanding correct that snapshot was for testing purposes only?

Reply:
The version will be changed while the releasing procedure.